File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/sso/LoginLdapReqVo.java
package org.javamaster.invocationlab.admin.model.sso;

import lombok.Data;

@Data
public class LoginLdapReqVo {
    private String account;
    private Integer pwdSwitch;
    private String appType;
    private String clientType;
    private String password;
    private String deviceNum;
    private String manuFacturer;
    private Integer accountType;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/sso/LoginLdapReqVo.kt
package org.javamaster.invocationlab.admin.model.sso

import org.javamaster.invocationlab.admin.annos.AllOpen


@AllOpen
class LoginLdapReqVo {
    var account: String? = null
    var pwdSwitch: Int? = null
    var appType: String? = null
    var clientType: String? = null
    var password: String? = null
    var deviceNum: String? = null
    var manuFacturer: String? = null
    var accountType: Int? = null
}
